### ISSUE TITLE
browser extension which visually mutes or hides non-focusable elements

### ISSUE THREAD

**Issue by techieshark** [techieshark/ideas#50]
For web accessibility development, it'd be nice to glance at a page and visually check that everything I think should be focusable is. 

For example if there is a button implemented as 

```
<a onclick="something;">not really a button</a>
```

It should fade or disappear when enabling this browser extension / option.
